docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   postgres:
     image: postgres:9.6
     ports:
-      - "5432"
+      - "5439"
```

Comment:
This will require an additional change to work. We can map the internal container port `5432` to the host port `5439` with the syntax `"5439:5432"`.

Reply:
Not with the docker.env change that makes sure the client gets build with the correct env values too though?

Reply:
I'm not sure what you mean by that